### PR TITLE
Genetics study-locus boilerplate code for util and summary component 

### PR DESCRIPTION
**Pull request by chinmehta**
[x] Study locus data transformation functions moved to be a separate util files
[x] Moved common functions in util from index.js to common.js file.
[x] Created boiler plate component code for summary component in study locus page.
[x] Part of issue#: https://github.com/opentargets/issues/issues/2662

[ ] TODO 1: Refactor existing functions in util to reduce the complexity of data transformation

### CODE REVIEW

**Review by carcruz — DISMISSED** [review]
Please create a branch named `study-locus-refactor` from `main` and use it as the target of this PR

File: apps/genetics/src/utils/studyLocus.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      data[`gwasCredibleSet__${study.studyId}__${indexVariant.id}`];
+    if (selection === undefined) return [];
+    return selection
+      .map(flattenPosition)
+      .filter((d) => (credSet95Value === '95' ? d.is95CredibleSet : true));
+};
+
+const flattenPosition = ({ tagVariant, postProb, is95, is99, ...rest }) => ({
+    tagVariant,
+    position: tagVariant.position,
```
[the remark this answers is not in the document]
Comment (chinmehta) [reply apps/genetics/src/utils/studyLocus.js:161]: Done in sub PR

File: apps/genetics/src/utils/index.js [hunk 2]
```diff
@@ -6,23 +6,18 @@ export function sanitize(str) {
   return str.replace(/[^a-zA-Z0-9]/g, '');
 }
```
[the remark this answers is not in the document]
Comment (chinmehta) [reply apps/genetics/src/utils/index.js:8]: Done in sub PR

**Comment by carcruz** [comment]
Please "Squash" the commits to reduce the amount in the PR

**Review by carcruz — APPROVED** [review]
🤞

**Pull request merged**